File: n8n-frontend/test/validate-mongo.py
from pymongo import MongoClient
from pymongo.errors import ConnectionFailure

uri = "mongodb+srv://aswalh0707:ASXMZYa5r2KkPHJO@automation.sfe092t.mongodb.net/?retryWrites=true&w=majority&appName=automation"

try:
    client = MongoClient(uri, serverSelectionTimeoutMS=5000)
    client.admin.command('ping')
    print("✅ Connection successful!")
except ConnectionFailure as e:
    print("❌ Connection failed:", e) 

File: n8n-frontend/test/validate-mongo.js
const { MongoClient } = require('mongodb');

const uri = 'mongodb+srv://aswalh0707:ASXMZYa5r2KkPHJO@automation.sfe092t.mongodb.net/?retryWrites=true&w=majority&appName=automation';

async function testConnection() {
  const client = new MongoClient(uri, { serverSelectionTimeoutMS: 5000 });
  try {
    await client.connect();
    await client.db().admin().ping();
    console.log('✅ Connection successful!');
  } catch (err) {
    console.error('❌ Connection failed:', err.message);
  } finally {
    await client.close();
  }
}

testConnection(); 